src/Microsoft.AspNet.Hosting/HostingContext.cs

```diff
@@ -17,7 +17,9 @@ public class HostingContext
         public IApplicationBuilder Builder { get; set; }
 
         public string ApplicationName { get; set; }
-        public string WebRootPath { get; set; }
+        // REVIEW: BasePath and WebRootPath are basically setting same thing?
+        public string ApplicationBasePath { get; set; }
+        //public string WebRootPath { get; set; }
```

Comment:
actually, not sure why this class is changing at all.  seems unrelated to avoiding creation of `IApplicationEnvironment` implementations in test code.